### DISCUSSION TITLE
Additional height scales similar to width-X/Y

### DISCUSSION THREAD

**Discussion by amankkg** [tailwindlabs/tailwindcss#2062] [Ideas]
Hi! I wonder why there is a kind of asymmetry between available width and height scales?

There is a similar issue https://github.com/tailwindcss/tailwindcss/issues/191 saying that file size is a concern.

Is there any other motivation to not to include it in default config?

### Width scales missing for height
Class | Properties
-- | --
.w-1/2 | width: 50%;
.w-1/3 | width: 33.333333%;
.w-2/3 | width: 66.666667%;
... | ...
.w-9/12 | width: 75%;
.w-10/12 | width: 83.333333%;
.w-11/12 | width: 91.666667%;

**Comment by estevanmaito** [comment]
This issue (even though the title only says width) also mentions height https://github.com/tailwindlabs/tailwindcss/issues/1917

There's a discussion going on there too.
